src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card">
+      <div class="card__product" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__photo"
+        >
+        <div class="card__name">
+          Моноблок APPLE A1419 iMac<br> 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="card__id">
+          Код товара: 195434
+        </div>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="star star--1"></div>
```

Comment:
if stars is a separate block then make each star its child
also rename modifier according to BEM

```suggestion
          <div class="stars">
            <div class="stars__star stars__star--1"></div>
```